.bluemix/pipeline.yml

```diff
@@ -86,34 +86,34 @@ stages:
       
       # Tracking Metrics
       if [ "NO" != "$DISABLE_METRIC" ]; then
-      echo "============================================================="
-      echo "Adding deployment data to metrics tracker... "
-      echo "============================================================="
-      space_id=$(cf space ${CF_SPACE} --guid)
-      service_id=$(cf service "Bot Asset Exchange Workspaces" --guid)
-      config='{"repository_id": "chatbot-deployer",
-             "target_runtimes": [],
-             "target_services": ["Conversation"],
-             "event_id": "web", 
-             "event_organizer": "dev-journeys"}'
-      DATE=$(date '+%Y-%m-%d %H:%M:%S')
-      data=$(cat <<-EOF
-      {
-      "space_id": "$space_id",
-      "service_id": "$service_id",
-      "date_sent":"$DATE",
-      "bot_name": "$CHATBOT_NAME_RAW",
-      "config": $config
-      }
+        echo "============================================================="
+        echo "Adding deployment data to metrics tracker... "
+        echo "============================================================="
+        space_id=$(cf space ${CF_SPACE} --guid)
+        service_id=$(cf service "Bot Asset Exchange Workspaces" --guid)
+        config='{"repository_id": "chatbot-deployer",
+               "target_runtimes": [],
+               "target_services": ["Conversation"],
+               "event_id": "web", 
+               "event_organizer": "dev-journeys"}'
+        DATE=$(date '+%Y-%m-%d %H:%M:%S')
+        data=$(cat <<-EOF
+        {
+        "space_id": "$space_id",
+        "service_id": "$service_id",
+        "date_sent":"$DATE",
+        "bot_name": "$CHATBOT_NAME_RAW",
+        "config": $config
+        }
       EOF
-      )
+        )
 
-      callback=$(curl -d "$data" -H "Content-Type: application/json" -X POST $TRACKER_URL)
-      if [[ "$callback" == *"true"* ]]; then 
-        echo "metrics sent successfully"
-      else
-        echo "metrics server error. Skipping..."
-      fi
+        callback=$(curl -d "$data" -H "Content-Type: application/json" -X POST $TRACKER_URL)
+        if [[ "$callback" == *"true"* ]]; then 
+          echo "metrics sent successfully"
+        else
+          echo "metrics server error. Skipping..."
+        fi
       
```

Comment:
i was also thinking you could add an else here and echo a message that the user has opted out of the metrics tracker